src/preprocess.py

```diff
@@ -575,6 +578,17 @@ def add_openai_bpe_vocab(vocab, namespace='openai_bpe'):
     vocab.add_token_to_namespace(utils.SOS_TOK, namespace)
     vocab.add_token_to_namespace(utils.EOS_TOK, namespace)
 
+
+def add_wsj_vocab(vocab, data_dir, namespace='tokens'):
+    '''Add WSJ vocabulary for PTB parsing models.'''
+    wsj_vocab_path = os.path.join(data_dir, 'WSJ/tokens.txt')
```

Comment:
Make sure there are instructions for how to get or create this file.

Reply:
I added some instructions. Although I would eventually like to upload the WSJ data folder (including the tokens.txt vocab file) in the cloud, l

Reply:
Reasonable. Two nits: Jiant => jiant, and capitalize 'run'.

Reply:
Done

Reply:
Where did the instructions go?

Reply:
They are there: https://github.com/jsalt18-sentence-repl/jiant/blob/86796bad424ce4e7fbe722cc174a4535b7adb0ec/src/preprocess.py